### PR TITLE
New Bitrix24 connector docs

### PR DESCRIPTION
**Pull request by joshuamullings**

### CODE REVIEW

File: pages/connector-authentication/bitrix24.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+6. Select **Save**.
+
+<a name="get-webhook-id"></a>
+
+#### Get webhook ID
+
+In the **Webhook to call REST API** field:
+
+1. Get the URL.
+2. Select just the random 16 character alphanumeric string. This is the webhook ID. For example, if the URL is `https://my.bitrix24instance.co.uk/rest/1/j2zdj8r9h0fdgsb3/` then the webhook ID is `j2zdj8r9h0fdgsb3`.
```
Comment (erin-lw) [inline pages/connector-authentication/bitrix24.md:33]: 1. how get URL?
2. select = click in our docs, 'make note of' or similar is probably a better way of doing it :)

File: pages/connector-authentication/bitrix24.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+Cyclr requires the following permissions for the Bitrix24 connector to function correctly:
+
+- `CRM (crm)`
+- `Users (user)` 
+
+Under the **Assign permissions** heading for each permission:
+
+1. Select **+ select**.
+2. Enter the required permission.
```
Comment (erin-lw) [inline pages/connector-authentication/bitrix24.md:47]: what does this mean? are they searching for it or entering a value?

Comment (joshuamullings) [reply pages/connector-authentication/bitrix24.md:47]: They would need to search for each of the permissions given in the list just above these instructions:
image
image

File: pages/connector-authentication/bitrix24.md (hunk 1 above)
Comment (joshuamullings) [reply pages/connector-authentication/bitrix24.md:33]: They're taking the j2zdj8r9h0fdgsb2 bit from this URL displayed on the page:
image

Does this sound better?
```suggestion
1. Make a note of the webhook ID. This is the random 16 character alphanumeric string that makes up part of the URL. For example, if the URL is `https://my.bitrix24instance.co.uk/rest/1/j2zdj8r9h0fdgsb3/` then the webhook ID is `j2zdj8r9h0fdgsb3`.
```

File: pages/connector-authentication/bitrix24.md (hunk 2 above)
Comment (erin-lw) [reply pages/connector-authentication/bitrix24.md:47]: A missing comma can cause me much confusion! :D

Under the **Assign permissions** heading, to add each permission:

**Review by erin-lw — APPROVED** [review]

**Pull request merged**